.travis.yml

```diff
@@ -19,9 +19,6 @@ script:
   - export PATH=/snap/bin:$PATH
   - sudo snapcraft cleanbuild
   - sudo cp *.snap "$(echo "$TRAVIS_REPO_SLUG" | sed -e 's|.*/\(.*\)|\1|')-pr$TRAVIS_PULL_REQUEST.snap"
-after_success:
-  - sudo snap install transfer
-  - timeout 180 sudo /snap/bin/transfer "$(echo "$TRAVIS_REPO_SLUG" | sed -e 's|.*/\(.*\)|\1|')-pr$TRAVIS_PULL_REQUEST.snap"
```

Comment:
why did you delete this?

Reply:
Transfer.com is shutting down

Reply:
oh, is it? hmm @popey what should we do about that?

Reply:
the transfer snap uses transfer.sh. Is that different to transfer.com or did you typo? Can you point me to the announcement that transfer.sh is shutting down because I can't seem to find any reference (I'm sure I'm failing at google. my fault, not yours :-) )

Reply:
aha. there's a message on their homepage:

```plain
Don’t Panic! transfer.sh will live on!
After running and supporting transfer.sh for 4 years on my own, I’m happy to announce we are partnering with Storj Labs to keep the project going. From day one, the transfer.sh code has been open source. Storj has a commitment to open source sustainability and reached out to help us find a way to keep our project alive. Stay tuned for updates on the partnership with Storj, but for now, please continue to enjoy the service!
Blue skies, Remco Verhoef
```

Looks like we're ok to continue using it :-)

Reply:
you might not, but we do to test pull requests before merging.